packages-proprietary/tokens/src/components/ams/progress-list.tokens.json

```diff
@@ -0,0 +1,38 @@
+{
+  "ams": {
+    "progress-list": {
+      "step": {
+        "padding-block-end": { "value": "{ams.space.l}" }
+      },
+      "progress": {
```

Comment:
We use ‘indicator’ in various places; let’s do so here (and in the CSS) as well.